src/components/CardComponent.jsx

```diff
@@ -1,27 +1,34 @@
-import React, { Component } from 'react'
-import { Card } from 'semantic-ui-react'
-
-class CardComponent extends React.Component {
-    statistics = this.props.statistics;
-
-    render() {
-        let cards = this.statistics.map((value, index) => {
-            return (
-                <Card key={index}>
-                    <Card.Content>
-                        <Card.Header>{value.value}</Card.Header>
-                        <Card.Description>{value.label}</Card.Description>
-                    </Card.Content>
-                </Card>
-            )
-        });
-
-        return (
-            <Card.Group centered>
-                {cards}
-            </Card.Group>
-        )
-    }
-}
-
-export default CardComponent;
+import React, { Component } from 'react';
+import { Card } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+class CardComponent extends Component {
+    statistics = this.props.statistics;
+
+    render() {
+      const cards = this.statistics.map((value, index) => (
+        <Card key={index}>

```

Comment:
As per houndci comment about `Do not use Array index in keys react/no-array-index-key`, please use a different key value. 

`react/no-array-index-key` discourages use of index as keys as it may result in having same keys on a page, resulting to React's duplicate key warning and having only one element sharing the key being displayed on the DOM, while the rest are forgone. If the array value doesn't have any `id` to use, you can make the key unique by extrapolating it with something else. For this case, you can do something like:
```
 key={`${value.value}-${index}`}
```